runtime/bcutil/ROMClassBuilder.cpp

```diff
@@ -1237,6 +1237,12 @@ ROMClassBuilder::computeExtraModifiers(ClassFileOracle *classFileOracle, ROMClas
 	if (classFileOracle->isValueBased()) {
 		modifiers |= J9AccClassIsValueBased;
 	}
+	
+	if (classFileOracle->hasIdentityInterface()
+		|| classFileOracle->needsIdentityInterface()
+	) {
+		modifiers |= J9AccClassIllegalSuperOfValueType;
```

Comment:
Can you rename this flag to `J9AccClassHasIdentity`. Same with the macros.

Reply:
Done.